src/Algebra/Consequences/Propositional.agda

```diff
@@ -100,31 +100,41 @@ module _ {_∙_ : Op₂ A} where
 
 module _ {_∙_ : Op₂ A} where
 
-  comm+assoc⇒middleFour : Commutative _∙_ → Associative _∙_ →
+  comm∧assoc⇒middleFour : Commutative _∙_ → Associative _∙_ →
                           _∙_ MiddleFourExchange _∙_
-  comm+assoc⇒middleFour = Base.comm+assoc⇒middleFour (cong₂ _∙_)
+  comm∧assoc⇒middleFour = Base.comm∧assoc⇒middleFour (cong₂ _∙_)
 
-  identity+middleFour⇒assoc : {e : A} → Identity e _∙_ →
+  identity∧middleFour⇒assoc : {e : A} → Identity e _∙_ →
                               _∙_ MiddleFourExchange _∙_ →
                               Associative _∙_
-  identity+middleFour⇒assoc = Base.identity+middleFour⇒assoc (cong₂ _∙_)
+  identity∧middleFour⇒assoc = Base.identity∧middleFour⇒assoc (cong₂ _∙_)
 
-  identity+middleFour⇒comm : {_+_ : Op₂ A} {e : A} → Identity e _+_ →
+  identity∧middleFour⇒comm : {_+_ : Op₂ A} {e : A} → Identity e _+_ →
                              _∙_ MiddleFourExchange _+_ →
                              Commutative _∙_
-  identity+middleFour⇒comm = Base.identity+middleFour⇒comm (cong₂ _∙_)
+  identity∧middleFour⇒comm = Base.identity∧middleFour⇒comm (cong₂ _∙_)
 
 ------------------------------------------------------------------------
 -- Without Loss of Generality
 
 module _ {p} {P : Pred A p} where
 
-  subst+comm⇒sym : ∀ {f} (f-comm : Commutative f) →
+  subst∧comm⇒sym : ∀ {f} (f-comm : Commutative f) →
                    Symmetric (λ a b → P (f a b))
-  subst+comm⇒sym = Base.subst+comm⇒sym {P = P} subst
+  subst∧comm⇒sym = Base.subst∧comm⇒sym {P = P} subst
 
   wlog : ∀ {f} (f-comm : Commutative f) →
          ∀ {r} {_R_ : Rel _ r} → Total _R_ →
          (∀ a b → a R b → P (f a b)) →
          ∀ a b → P (f a b)
   wlog = Base.wlog {P = P} subst
+
+
+------------------------------------------------------------------------
+-- DEPRECATED NAMES
+------------------------------------------------------------------------
+-- Please use the new names as continuing support for the old names is
+-- not guaranteed.
+
+-- Version 2.0
+-- See all the names deprecated in Algebra.Consequences.{Base|Setoid}
```

Comment:
We still need to create deprecated versions of the old names here, as they have different arguments to the ones exported by `Algebra.Consequences.Setoid`.

Reply:
Ah. For some reason, I thought that it was enough for the use of deprecated *names* to trigger the warnings, so that that behaviour could 'delegate' to `...Setoid`. More confusion/ignorance/uncertainty on my part. I'll fix when I'm awake. I'm not really awake now... it's my middle of the night insomnia phase.

Reply:
Done now, again I hope successfully.

Reply:
Thanks, that looks great!